tavern/testutils/pytesthook/item.py

```diff
@@ -29,13 +31,23 @@ class YamlItem(pytest.Item):
         spec (dict): The whole dictionary of the test
     """
 
+    py2_warned = False
+
     def __init__(self, name, parent, spec, path):
         super(YamlItem, self).__init__(name, parent)
         self.path = path
         self.spec = spec
 
         self.global_cfg = {}
 
+        if sys.version_info < (3, 0, 0):
+            if not YamlItem.py2_warned:
+                warnings.warn(
+                    "Tavern will drop support for Python 2 in a future release, please switch to using Python 3",
```

Comment:
Our trigger for dropping will be when pytest stops supporting python 2 right? should we mention that and/or link to the pytest roadmap https://docs.pytest.org/en/latest/py27-py34-deprecation.html

Reply:
Done